plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -106,6 +176,68 @@ public void testDescribeTable()
         assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test
+    @Override
+    public void testPredicateReflectedInExplain()
+    {
+        // The format of the string representation of what gets shown in the table scan is connector-specific
+        // and there's no requirement that the conform to a specific shape or contain certain keywords.
```

Comment:
I need a little help regarding the `TODO` because I don't know the convention regarding such things in the project.
Can you describe exactly what do you expect me to do here?

Reply:
@findinpath When adding a TODO in the code we generally create an issue in GitHub describing what needs to be done to be able to remove the TODO. And then in the TODO comment we add a link to the GitHub issue - that way future readers (or the person closing the issue) can update the code and remove TODO comments.

So you can just create a GitHub issue like `Provide consistent representation of pushed-down operations in EXPLAIN output` (and maybe add a link to https://github.com/trinodb/trino/pull/7609#discussion_r616148357).

Reply:
I hope I got it right.